DigitalLearningSolutions.Web/Controllers/TrackingSystem/Delegates/CourseDelegatesController.cs

```diff
@@ -45,47 +48,65 @@ ICourseDelegatesDownloadFileService courseDelegatesDownloadFileService
             string sortDirection = GenericSortingHelper.Ascending,
             string? existingFilterString = null,
             string? newFilterToAdd = null,
+            bool clearFilters = false,
             int page = 1
         )
         {
             sortBy ??= DefaultSortByOptions.Name.PropertyName;
             var newFilterString = FilteringHelper.GetFilterString(
                 existingFilterString,
                 newFilterToAdd,
+                clearFilters,
                 Request,
                 CourseDelegatesFilterCookieName,
                 CourseDelegateAccountStatusFilterOptions.Active.FilterValue
             );
 
             var centreId = User.GetCentreId();
             var adminCategoryId = User.GetAdminCourseCategoryFilter();
-            CourseDelegatesData courseDelegatesData;
 
             try
             {
-                courseDelegatesData =
-                    courseDelegatesService.GetCoursesAndCourseDelegatesForCentre(
-                        centreId,
-                        adminCategoryId,
-                        customisationId
-                    );
+                var courseDelegatesData = courseDelegatesService.GetCoursesAndCourseDelegatesForCentre(
+                    centreId,
+                    adminCategoryId,
+                    customisationId
+                );

```

Comment:
The rest of the stuff I've moved into the try relies on the courseDelegatesData, so I moved it all in rather than having an uninitialised one before the try. I don't know if there's any general preference for code style on this though?